src/query/service/src/pipelines/processors/transforms/transform_udf_script.rs

```diff
@@ -27,15 +31,15 @@ use databend_common_expression::FunctionContext;
 use databend_common_pipeline_transforms::processors::Transform;
 use databend_common_pipeline_transforms::processors::Transformer;
 use databend_common_sql::executor::physical_plans::UdfFunctionDesc;
+use databend_common_storage::DataOperator;
+use opendal::Operator;
 
 use crate::pipelines::processors::InputPort;
 use crate::pipelines::processors::OutputPort;
 use crate::pipelines::processors::Processor;
+
 pub struct TransformUdfScript {
     funcs: Vec<UdfFunctionDesc>,
-    js_runtime: Arc<arrow_udf_js::Runtime>,
```

Comment:
why not inject `py_runtime` and `wasm_runtime` here?

we don't need to build a new runtime for each input block.

Reply:
The initial aim was to inject the runtime instance as part of `TransformUdfScript::try_create()`. However, the `add_function_with_handler()` interface, which allows coupling a UDF function handler with the source code script, was only available for the `arrow-udf-js` and `arrow-udf-python` runtimes, not for `arrow-udf-wasm`. 

In the `arrow-udf-wasm` context, each UDF's source code module is directly linked to its runtime instance during creation (`Runtime::new()`). 

To ensure a uniform approach across all runtime environments, the decision was made to create runtime instances within the `TransformUdfScript::transform()` function.

This method provides a standardized way of handling UDFs Runtime::new()`). 

The performance overhead due to creating a runtime instance for each UDF descriptor within `TransformUdfScript::transform()` function is understandable. I will re-work on this and get back soon.

Reply:
We can introduce an enum to handle this.

```
enum ScriptRuntime { 
  Python(PyRuntime),
   WASM(WASMRuntime),
   JS(JsRuntime),
}

pub struct TransformUdfScript {
    funcs: Vec<UdfFunctionDesc>,
    runtime: Arc<ScriptRuntime>,
  ..
} 

impl ScriptRuntime {
 ...
}
```

Reply:
Code changes completed and ready for review.